server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,51 @@
+package psql
+
+import (
+	"time"
+
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+type rawTime []byte
+
+func (t rawTime) Time() (time.Time, error) {
```

Comment:
I think you can delete line 10 to line 14. The code works without it and you're not using it.

Reply:
ok